### ISSUE TITLE
Install Cogmind with "Steam Play for all other titles" & "Steam Linux Runtime" (default) results in unplayable installation

### ISSUE THREAD

**Issue by winny-** [ValveSoftware/steam-for-linux#6803]
#### Your system information

* Steam client version (build number or date): Dec 16 2019, at 16:49:05
* Distribution (e.g. Ubuntu): Gentoo
* Opted into Steam client beta?: No
* Have you checked for system updates?: N/A
* [Steam System Information](https://gist.github.com/winny-/5593ea932930bb2f30692967fa014deb)

#### Please describe your issue in as much detail as possible:

When installing Cogmind with "Enable Steam Play for all other titles" enabled (Settings → Steam Play), and the "Run other titles with" dropdown set to the default - "Steam Linux Runtime", results in Steam to download the game instantly. If I open up the game directory, it is an empty folder. 

Upon launching Cogmind I get error message:
```
An error occured while updating Cogmind (missing executable): 
/home/winston/.local/share/Steam/steamapps/common/Cogmind/COGMIND.exe
```

If I go in and change the "Run other titles with" to anything other than "Steam Linux Runtime" Cogmind will install properly. Alternatively if I change Cogmind's Steam Play tool, that also fixes the problem (Right click on Cogmind → Properties → General → [x] Force the use of a specific Steam Play compatibility tool, choose from dopdown).

In other words, Steam is permitting me to install a game with an invalid Steam Play configuration, causing much confusion and asking questions.

#### Steps for reproducing this issue:

1. Install Steam on Linux
2. Enable Steam Play for other titles, leave the default tool to "Steam Linux Runtime"
3. Install Cogmind
4. Try to launch Cogmind

**Comment by kisak-valve** [comment]
Hello @winny-, by selecting Steam Linux Runtime, you've told Steam to not use Proton with the Windows game on Linux. This is a known limitation with the initial deployment of the tool and is being tracked at https://github.com/ValveSoftware/steam-runtime/issues/200.

Closing as a duplicate.
